### PR TITLE
Added some 'base' docker files for clusternator projects.

### PR DESCRIPTION
**Pull request by SethDavenport**
The idea is that if you're using the node/express stack you don't want to do all the apt-get stuff everytime you build. Same thing for downloading and installing node.

So there's a base 'ubuntu + apt-get + set up swuser' image
Then there are node 4 and 5 images that depend on the base one
And finally your clusternated project should depend on one of the node ones.

Then all that needs to be done in CI is npm install, serve.sh.

### CODE REVIEW

**Comment by SethDavenport** [comment]
@bennett000 @rafkhan @odziem

**Comment by SethDavenport** [comment]
Right now these images are in my personal dockerhub account.  However if you guys like this approach I'll ask Yuri to let me put them in a rangle account instead.

If I wanted to hook these up to the menu you get from `clusternator init` what part of the codebase should I look at?

**Comment by SethDavenport** [comment]
This helps us by defining a 'Base image' that we can add stuff to, so we can avoid every project having to update their docker files when things change.

**Comment by bennett000** [comment]
This all makes sense.

I was thinking that #213 and/or the idea of having an _organization_ level private docker repo on AWS might be a good place to put common images.

**Comment by odziem** [comment]
Looks good. Wouldn't the apt-get stuff get cached thanks to Docker's layering anyway?

**Comment by SethDavenport** [comment]
@odziem : yes, except as it turns out on circle-ci:

> Docker images aren't cached automatically. At the moment, we have no good method to cache them although we are trying to find a technical solution.

(https://circleci.com/docs/docker)

However the base image approach has other benefits: we can tune them centrally and projects can consume the updates more easily.

For example for the node one, I'm thinking eventually of fronting it with nginx and allowing people to put in custom configs for API proxies, etc.

This kind of flexibility will be good going forward.

**Comment by SethDavenport** [comment]
@bennett000 : I think the base images should be public, because clusternator is open source.  It would be annoying for a 3rd party to spin up a clusternator instance only to find that the base images aren't available.

**Comment by bennett000** [comment]
@SethDavenport sorry I misunderstood.  Clusternator images can definitely be public.  I was thinking of other base images that might be more tuned to a specific project.

**Comment by SethDavenport** [comment]
Cool. As soon as I get permissions I'll push the base images to rangle/ on docker hub.

**Comment by odziem** [comment]
@SethDavenport Ah ok. Agreed. :+1:

File: src/api/0.1/project-fs/skeletons/create-interactive-questions.js [hunk 1]
```diff
@@ -166,7 +166,7 @@ function projectInit(defaults) {
     {
       type: 'list',
       name: 'backend',
-      choices: ['static', 'node'],
+      choices: ['static', 'node (long-term-support)', 'node (stable)'],
       message: 'Choose a backend'
     },
     {
```
Comment (bennett000) [inline src/api/0.1/project-fs/skeletons/create-interactive-questions.js]: I'm really looking forward to fixing this whole file at some point.  It sort of made sense in early days but as things have shifted around it's become confusing.

**Comment by SethDavenport** [comment]
The base images are now in `rangle` dockerhub.

I bumped the npm version to 0.5 and will publish soon.

**Pull request merged**